src/Commands/utilities/commands.js

```diff
@@ -46,24 +51,25 @@ module.exports = {
 			economyArray = [];
 			otherArray = [];
 			setupArray = [];
+			
 			const promiseArray = client.commands.map(async (c) => {
 				if (c.type === "Utility") {

```

Comment:
javascript's `switch () { case "...": ... }` might be a good thing in such cases, read more on the mdn docs https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch